frb_codegen/src/generator/rust/ty_delegate.rs

```diff
@@ -140,6 +141,26 @@ impl TypeRustGeneratorTrait for TypeDelegateGenerator<'_> {
         }
     }
 
+    fn deallocate_funcs(
```

Comment:
Given that all types have added dealloc funcs, it looks critical we have some tests to cover all these cases.

Thus, we may need to add tests in frb_dart, such that we ensure each dealloc are run.

Reply:
validation delegated to rust wire2api